src/transform.ts

```diff
@@ -390,6 +390,19 @@ export interface FoldTransform {
   as?: [FieldName, FieldName];
 }
 
+export interface ExtentTransform {
+  /**
+   * The field of which to get the extent.
+   */
+  extent: FieldName;
+
+  /**
+   * The output parameter produced by the extent transform.
+   * __Default value:__ `"extent"`
```

Comment:
Can there really be a default? Would that mean you pass undefined or null? Maybe we should not make this optional.

Reply:
I agree. Updated to fix this.